app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -43,7 +42,10 @@ echo "the grub2_chainload template."
 echo
 echo "(*) grub2-efi-x64-2.02-122.el8 (upstream doesn't have the patches yet)"
 echo
-#connectefi scsi
+<%=
+connectefi_option = @host && host_param('pxegrub_connectefi_option')
+"connectefi #{connectefi_option}" if connectefi_option
```

Comment:
This doesn't account for empty strings, so there's no way to go from a global default to turning it off.

Reply:
You can always set it to `false` I suppose...